packages/cozy-intent/package.json

```diff
@@ -10,9 +10,9 @@
     "url": "https://github.com/cozy/cozy-libs/issues"
   },
   "dependencies": {
-    "cozy-device-helper": "^1.16.0",
+    "cozy-device-helper": "1.16.0",
     "post-me": "0.4.5",
-    "typescript": "^4.5.2"
+    "typescript": "4.5.4"
```

Comment:
Is it really a dep? It can't be a devDep? cc @trollepierre

Reply:
@Crash-- it was initially, but cozy-libs failed to build. I did not understand why. I guess I should take time to see what's up, but I thought it was not the most important thing right now. I should create an issue